x-pack/test/apm_api_integration/tests/throughput/index.ts

```diff
@@ -0,0 +1,150 @@
+import archives_metadata from '../../common/fixtures/es_archiver/archives_metadata';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { registry } from '../../common/registry';
+import { roundNumber } from '../../utils';
+
+type ThroughputReturn = APIReturnType<'GET /api/apm/services/{serviceName}/throughput'>;
+type ServiceReturn = APIReturnType<'GET /api/apm/services'>;
+type TransactionsDetailsReturn = APIReturnType<'GET /api/apm/services/{serviceName}/transactions/groups/main_statistics'>;
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const apmApiSupertest = createApmApiSupertest(getService('supertest'));
+
+  const archiveName = 'apm_8.0.0';
+  const metadata = archives_metadata[archiveName];
+
+  let throughputResponse: ThroughputReturn;
+  let serviceResponse: ServiceReturn;
+  let transactionsDetailsReturn: TransactionsDetailsReturn;
+
+  const deltaAsMinute =
+    (new Date(metadata.end).valueOf() - new Date(metadata.start).valueOf()) / 1000 / 60;
+
+  function callAPIs({ processorEvent }: { processorEvent: 'transaction' | 'metric' }) {
+    return Promise.all([
+      apmApiSupertest({
+        endpoint: 'GET /api/apm/services',
+        params: {
+          query: {
+            kuery: `service.name : "opbeans-java" and processor.event : "${processorEvent}"`,
+            start: metadata.start,
+            end: metadata.end,
+            environment: 'ENVIRONMENT_ALL',
+          },
+        },
+      }),
+      apmApiSupertest({
+        endpoint: 'GET /api/apm/services/{serviceName}/throughput',
+        params: {
+          path: {
+            serviceName: 'opbeans-java',
+          },
+          query: {
+            kuery: `processor.event : "${processorEvent}"`,
+            start: metadata.start,
+            end: metadata.end,
+            transactionType: 'request',
+            environment: 'ENVIRONMENT_ALL',
+          },
+        },
+      }),
+      apmApiSupertest({
+        endpoint: `GET /api/apm/services/{serviceName}/transactions/groups/main_statistics`,
+        params: {
+          path: {
+            serviceName: 'opbeans-java',
+          },
+          query: {
+            kuery: `processor.event : "${processorEvent}"`,
+            start: metadata.start,
+            end: metadata.end,
+            transactionType: 'request',
+            environment: 'ENVIRONMENT_ALL',
+            latencyAggregationType: 'avg' as LatencyAggregationType,
+          },
+        },
+      }),
+    ]);
+  }
+
+  registry.when(
+    'Throughput value across apis',
+    { config: 'basic', archives: [archiveName] },
+    () => {
+      describe('with kql filter to force transaction-based UI', () => {
+        before(async () => {
+          const [
+            serviceApiResponse,
+            serviceThroughputApiResponse,
+            transactionsDetailsApiReturn,
+          ] = await callAPIs({ processorEvent: 'transaction' });
```

Comment:
`serviceName` should be a param and to avoid duplications between the tests I think the calculations should be made within `callAPIs` (renamed to `getThroughputValues`)

```suggestion
          const {
            serviceInventoryThroughput,
            serviceTimeseriesAvgThroughput,
            transactionGroupsTableSumThroughput,
          } = await getThroughputValues({ processorEvent: 'transaction', serviceName: 'opbeans-java' });
```